src/NBitcoin/NoSqlBlockRepository.cs

```diff
@@ -16,9 +16,9 @@ public NoSqlBlockRepository(NoSqlRepository repository)
             this.repository = repository;
         }
 
-        public NoSqlBlockRepository()
-            : this(new InMemoryNoSqlRepository())
-        {
+		public NoSqlBlockRepository(NetworkOptions options = null)
```

Comment:
Maybe I will spend a PR to remove this `NoSqlBlockRepository` code, its used in the IndexStore code which we don't use in the fullnode as we store the full chain in dbreeze instead.

Reply:
Ok